buildpack.md

```diff
@@ -95,19 +95,23 @@ Executable: `/bin/build <platform[AR]> <cache[EC]> <launch[EI]>`, Working Dir: `
 | [exit status]                 | Success (0) or failure (1+)
 | `/dev/stdout`                 | Logs (info)
 | `/dev/stderr`                 | Logs (warnings, errors)
-| `<cache>/<layer>/bin/`        | Binaries for subsequent buildpacks
-| `<cache>/<layer>/lib/`        | Libraries for subsequent buildpacks
-| `<cache>/<layer>/include/`    | C/C++ headers for subsequent buildpacks
-| `<cache>/<layer>/pkgconfig/`  | Search path for pkg-config
+| `<cache>/<layer>/bin/`        | Binaries for subsequent buildpacks (added to `PATH` in build phase)
```

Comment:
The env vars are already mentioned in the environment section below, and specifying this again up here makes the table wrap. Could we link to that section instead?